lib/raidenclient/RaidenClient.ts

```diff
@@ -41,6 +41,12 @@ class RaidenClient extends SwapClient {
   private host: string;
   private disable: boolean;
 
+  // TODO: Populate the mapping from the database (Currency.decimalPlaces).
```

Comment:
Maybe open an issue for this `TODO` so that you don't forget about it

Reply:
I think #1054 is tracking this.